app/controllers/hostgroups_controller.rb

```diff
@@ -77,9 +76,9 @@ def destroy
 
   def current_parameters
     Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
+      hostgroup = Hostgroup.authorized(:view_hostgroups).find(params['hostgroup_id'])
```

Comment:
I don't think this works with a new host group, since there won't be an ID and we can't look them up.  The old code used to only assume that the parent, if specified, exists.